_includes/getting-started.html

```diff
@@ -109,126 +127,123 @@ <h4 style="font-size: 1.5em;">Using Helm</h4>
+          <div class="hidediv">
+            <div class="innerdiv">
+              <p>Download and unzip <span style="color: rgb(117, 250, 255);">mesheryctl</span> from the <a
+                  href="https://github.com/layer5io/meshery/releases/tag/v0.3.9">Meshery releases</a> page. Add <span
+                  style="color: rgb(117, 250, 255);">mesheryctl</span> to your PATH for ease of use. Then, execute:</p>
+              <div id="start" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
+                <div style="width:auto; height:30px; padding:0px; position:relative;">
+                  <pre><code>   $ ./mesheryctl start
               </code></pre>
-              <!-- copy to clipboard -->
-              <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4"
-                style="position: absolute;top:-14px; right: -1px; border: 0; 
+                  <!-- copy to clipboard -->
+                  <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4" style="position: absolute;top:-14px; right: -1px; border: 0; 
                     cursor: pointer; background:#00a4ca; width:60px; height:44px; margin-bottom:35px; 
-                    border-radius: 2px; margin:0px 0px 880px 0px;"
-                data-clipboard-target="#start"
-                data-clipboard-text="./mesheryctl start">
-                <img src="/images/buttons/clippy.svg" style="width:20px; margin: 0; position: absolute; top: 50%; left: 54%; transform: translate(-50%, -50%);" />
-              </a>
+                    border-radius: 2px; margin:0px 0px 880px 0px;" data-clipboard-target="#start"
+                    data-clipboard-text="./mesheryctl start">
+                    <img src="/images/buttons/clippy.svg"
+                      style="width:20px; margin: 0; position: absolute; top: 50%; left: 54%; transform: translate(-50%, -50%);" />
+                  </a>
+                </div>
+              </div>
+
+            </div>
           </div>
         </div>
-          
-        </div>  
-  </div>
-</div>
-<hr>
-    <div class="login">
-      <h2 style="margin-top: 1em;"><strong>Login</strong></h2>
-    <h1 style="font-size: 80px;opacity: 0.2;margin-top: -95px;margin-left: -55px;"><strong>02</strong></h1>
-    <p style="margin-left:1em;">Access Meshery in your browser if you are using Kubernetes at <a href="http://127.0.0.1:9081" style="color: rgb(85, 217, 240);">http://127.0.0.1:9081</a>.</p>
-    </div>
-    
-    <!-- <h3 style="color:#111111;">Step 3: Configure and Enjoy Meshery</h3>
+        <hr>
+        <div class="login">
+          <h2 style="margin-top: 1em;"><strong>Login</strong></h2>
+          <h1 style="font-size: 80px;opacity: 0.2;margin-top: -95px;margin-left: -55px;"><strong>02</strong></h1>
+          <p style="margin-left:1em;">Access Meshery in your browser if you are using Kubernetes at <a
```

Comment:
<img width="1127" alt="Screenshot 2020-04-28 at 9 13 39 PM" src="https://user-images.githubusercontent.com/44091822/80508222-65d7df00-8995-11ea-9c0c-54100e66a5dc.png">

Was this included in this part of the issue or is it other? I am not sure if this is also an alignment issue.